src/include/enclave/lthread.h

```diff
@@ -278,12 +278,12 @@ extern "C"
      */
     int lthread_setspecific_remote(struct lthread* lt, long key, const void* value);
 
-    static void* lthread_getspecific(long key)
+    static inline void* lthread_getspecific(long key)
```

Comment:
Shouldn't the compiler figure this out by itself?

Reply:
Apparently not, it spits out a warning and it seems like static inline is the right way to define functions in headers. I'm not an expert on this though.